pkg/visor/remote_hypervisor.go

```diff
@@ -0,0 +1,23 @@
+package visor
+
+import (
+	"io/ioutil"
+	"net/http"
+)
+
+// FetchHvPk fetches the hypervisor public key from the ip:port passed to it.
+func FetchHvPk(ipPort string) (string, error) {
+	req, err := http.NewRequest(http.MethodGet, "http://"+ipPort, nil)
+	if err != nil {
+		return "", err
+	}
+	res, err := http.DefaultClient.Do(req)
```

Comment:
Maybe use `http.Get` right away? Also should close the body I believe.

Reply:
Basically this whole block of code could look like the example on `http` package documentation:

```
resp, err := http.Get("http://example.com/")
if err != nil {
	// handle error
}
defer resp.Body.Close()
body, err := io.ReadAll(resp.Body)
```

Reply:
`http.Get` cannot handle ports.

That is specifically why I had to do it this way.

Reply:
My bad.

Reply:
fixed here
[f4429dbfe922288c10a5ed49d9fec5bd7a36220f](https://github.com/skycoin/skywire/pull/1309/commits/f4429dbfe922288c10a5ed49d9fec5bd7a36220f)